Controller/RedirectRouteImportController.php

```diff
@@ -70,12 +70,19 @@ public function getLocale(Request $request)
      */
     public function postAction(Request $request)
     {
-        if (!$request->files->has('redirectRoutes')) {
+        if (!$request->files->has('files')) {
```

Comment:
Think we should configure this in the ToolbarAction as which name it is send to the endpoint.